### DISCUSSION TITLE
Stucked in the step 6 and 7 of android setup

### DISCUSSION THREAD

**Discussion by callmehanu** [CharlesChiuGit/Logseq-Git-Sync-101#32] [Q&A]
This is my current working path, I don't know if is the correct path, btw
![image](https://user-images.githubusercontent.com/13230502/230633140-ffd7c084-25ab-4ef6-a285-aec0f3c79472.png)

So I've tried to clone my repo but seems I don't have the  permissions
![image](https://user-images.githubusercontent.com/13230502/230635502-b0625de4-5384-43ee-a8f2-1bef420af734.png)

**Comment by callmehanu** [comment]
Sure I need to setup my git account in my phone but I don't clear how to

**Comment by charliie-dev** [comment]
Sorry about the late reply. Quite busy yesterday. 
To config your git info, follow step 5 in the Android part.

**Comment by callmehanu** [comment]
Hi @CharlesChiuGit... I followed all steps but anyway when I tried to clone my repo the console show me a permision denied error.
![image](https://user-images.githubusercontent.com/13230502/231639920-453a52b5-f69a-4a3c-9b56-3f1db90a0e1a.png)

Do I need to do any extra setup between my phone and my github account before try to clone my repo?

**Reply by charliie-dev** [reply]
Have u create and add the ssh key from your phone to your github account?

**Reply by callmehanu** [reply]
Sorry @CharlesChiuGit I didn't' realize in that.
By the way do you know how can create and ssh key in mobile? In this guide just have the case for MacOS, Windows and Linux.
https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent

**Comment by callmehanu** [comment]
Got it!
![image](https://user-images.githubusercontent.com/13230502/231642956-af5eb61b-88da-4456-905f-65f89e59b5b6.png)
In my ssh directory I already have this files.

**Reply by charliie-dev** [reply]
So it seems like u haven't create a ssh key on your phone.

**Reply by charliie-dev** [reply]
https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent

**Comment by callmehanu** [comment]
Finally I got the .pub file (id_ed25519) 
![image](https://user-images.githubusercontent.com/13230502/231652712-b2daecb9-a510-4a61-9a7e-df3400d83120.png)

but when I copy its content and add as new ssh key... github show me another error. 

![Frame 10003](https://user-images.githubusercontent.com/13230502/231653736-254d3ba9-b2eb-4241-ba03-285016b6b279.png)

**Reply by charliie-dev** [reply]
be sure to copy EVERYTHING in the pub file

**Comment by callmehanu** [comment]
Finally I added the sshkey with Github CLI!

**Reply by charliie-dev** [reply]
nice!

**Comment by callmehanu** [comment]
Btw... following the last steps, I've installed Termux Widget... but I didn't found the /shortcuts directory
![image](https://user-images.githubusercontent.com/13230502/231658199-ef377f01-9bae-47ce-97b2-301c9c4bd2cd.png)

**Reply by charliie-dev** [reply]
create one if it's not existed.
